simpleviewinc/tap-events-force
# Zen 452 remove test btn

**Ticket**: [issue_id](https://jira.simpleviewtools.com/browse/ZEN-452)

## Context
* access to the test button was removed last sprint, but we have lingering files from the test/demo container
*  Remove all traces of the demo container

## Goal (Updated 11/17/20)

* remove unused container/files relating to demo container
* update `rollup.config` to remove uneeded dependencies

## Updates

* removed a few uneeded files relating to demo/test container
* updated `tap.js` for the routes and entry point
* `rollup.config.js`
   - removed uneeded package.json deps
   - updated main file to cjs instead of esm

## Testing

* run `keg evf pack run package=docker.pkg.github.com/simpleviewinc/keg-packages/tap:zen-452-remove-test-btn`
* navigate to http://evf-zen-452-remove-test-btn.local.kegdev.xyz/
* verify that the path `/test` and `/demo` no longer exists

* while the tap is running, run `keg evf && yarn docker:build` on your local
* navigate to `tap-events-force/build/package.json`
* ensure the `dependencies` does NOT contain
   * 'axios',
   * 'react-native',
   * 'react-native-svg',

### Consumer
* pull this branch locally
* pull latest [keg-test-consumer](https://github.com/simpleviewinc/keg-test-consumer) `develop` branch
* start the consumer and sync with evf
    * `keg consumer start`
    * `keg consumer sync evf:install:start`
    * navigate to http://consumer-develop.local.kegdev.xyz/
* modify keg-test-consumer `App.js` line `24` adding the prop `disableDemo={false}` > Save
    ```Javascript
     return <Sessions 
       disableDemo={false}
       onSessionBookingRequest={onBookingRequest}
       onSessionWaitingListRequest={onWaitingRequest}
       sessionAgendaProps={testData} 
       ModalComponent={SessionsModal}
     />
    ```
* Verify on http://consumer-develop.local.kegdev.xyz/ , you don't seethe demo on the top left anymore
